scripts/stargz/setup_stock_only_stargz.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
```

Comment:
Would this script work both for multi-node and single-node setups? We need both.

Reply:
Yes. I have tested it in a 2 node cluster.